DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: trias
 Title: Process Data for the Project Tracking Invasive Alien Species
     (TrIAS)
-Version: 3.0.3
+Version: 3.0.4
```

Comment:
Personally I would bump to 3.1.0.

Reply:
Solved via https://github.com/trias-project/trias/pull/164/commits/c39947c69205130f39b9aa1be81f48769a8228a4